File: docs/breez-sdk/snippets/rust/src/fiat_currencies.rs
use anyhow::Result;
use breez_sdk_spark::BreezClient;

async fn list_fiat_currencies(client: BreezClient) -> Result<()> {
    // ANCHOR: list-fiat-currencies
    let response = client.fiat().currencies().await?;
    // ANCHOR_END: list-fiat-currencies

    Ok(())
}

async fn list_fiat_rates(client: BreezClient) -> Result<()> {
    // ANCHOR: list-fiat-rates
    let response = client.fiat().rates().await?;
    // ANCHOR_END: list-fiat-rates

    Ok(())
}

File: docs/breez-sdk/snippets/csharp/FiatCurrencies.cs
using Breez.Sdk.Spark;

namespace BreezSdkSnippets
{
    class FiatCurrencies
    {
        async Task ListFiatCurrencies(BreezClient client)
        {
            // ANCHOR: list-fiat-currencies
            var response = await client.Fiat().Currencies();
            // ANCHOR_END: list-fiat-currencies
        }

        async Task ListFiatRates(BreezClient client)
        {
            // ANCHOR: list-fiat-rates
            var response = await client.Fiat().Rates();
            // ANCHOR_END: list-fiat-rates
        }
    }
}
